src/services/http/httpService.js

```diff
@@ -76,6 +74,16 @@ function filtersToQuery(filters) {
         return response.json();
     },
 
+    getLocationsData : async (lat, lon, filters) => {
```

Comment:
name indicates that it gets data of the location which it does not do.

Reply:
also this function should probably end up somewhere else then in this file (this is rather 'non-business-logic' file)

Reply:
You're right. I'm closing this issue.